pkg/controller/workload/workload_processor.go

```diff
@@ -57,6 +56,8 @@ type Processor struct {
 	nodeName      string
 	WorkloadCache cache.WorkloadCache
 	ServiceCache  cache.ServiceCache
+	EndpointCache cache.EndpointCache
```

Comment:
why do you add it back>?

Reply:
I have using *cache.EndpointCache before, and found it may occur some segament error, so i decided to use stuct rather than pointer of it. EndpointCache is a new cache type introduced this time.